agent/build_session_test.go

```diff
@@ -142,7 +142,7 @@ func TestCleandirCommand(t *testing.T) {
 	}
 
 	for i, f := range matches {
-		actual := f[len(wd)+1:]
+		actual := f[len(wd) + 1:]
```

Comment:
Did you use gofmt format file? It is different format you're using with my gofmt result (on Mac OS).